src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,57 +1,129 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = (props) => {
+  const { addMovie } = props;
+
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isMovieLoading, setIsMovieLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const handleSearchBtn = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    setIsMovieLoading(true);
+
+    if (query.trim().length) {
```

Comment:
you need some `else` here, cause users will have an infinite load when trying to search some empty spaces
<img width="377" alt="image" src="https://user-images.githubusercontent.com/55802630/187190015-b2f0aec1-8fc4-4c2c-bcf7-c2d6c77d70bc.png">